rero_mef/api.py

```diff
@@ -57,18 +60,76 @@ def create(cls, data, id_=None, delete_pid=True,
             record.dbcommit(reindex)
         return record
 
+    @classmethod
+    def create_or_update(
+        cls,
+        data,
+        id_=None,
+        delete_pid=True,
+        dbcommit=False,
+        reindex=False,
+        **kwargs
+    ):
+        """Create or update agency record."""
+        pid = data['identifier_for_person']
+        agency_record = cls.get_record_by_pid(pid, with_deleted=False)
+
+        from .authorities.api import ViafRecord
+        viaf_record = ViafRecord.get_viaf_by_agency_pid(
+            pid, pid_type=cls.agency_pid_type, with_deleted=False
+        )
+        mef_action = agency_action = None
+        return_record = {}
+        if agency_record:
+            if viaf_record:
+                agency_record.update(data, dbcommit=dbcommit, reindex=reindex)
+                mef_action = 'update'
```

Comment:
status should be defined as a class such as:
```python
class MEFStatus(object):
    DISCARDED = 'discarded'
    CREATED = 'created
```
Can be used as `MEFStatus.CREATED`

Reply:
Done